integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/PetStoreTestCase.java

```diff
@@ -0,0 +1,206 @@
+    }
+
+    @Test(description = "Tests the building of pet store frontend image.")
+    public void buildFrontEnd() throws Exception {
+        build("pet-fe.bal", Constants.CELL_ORG_NAME, FRONTEND_IMAGE_NAME, VERSION,
+                Paths.get(CELLERY_SCENARIO_TEST_ROOT, "pet-store", "pet-fe").toFile().getAbsolutePath());
+    }
+
+    @Test(description = "Tests the running of pet store front end instance.")
+    public void runFrontEnd() throws Exception {
+        run(Constants.CELL_ORG_NAME, FRONTEND_IMAGE_NAME, VERSION, FRONTEND_INSTANCE_NAME, LINK, false,
+                600);
+    }
+
+    @Test(description = "Tests invoking of pet store web page.")
+    public void invoke() {
+        webDriver.get(Constants.DEFAULT_PET_STORE_URL);
+        String petAccessoriesHeader = webDriver.findElement(By.cssSelector("H1")).getText();
+        validateWebPage(petAccessoriesHeader, Constants.PET_STORE_WEB_CONTENT, "Pet store web page content is not " +
+                "as expected");
+    }
+
+    @Test(description = "This tests sign in for user Alice.")
+    public void signInAlice() throws InterruptedException {
+        this.signIn(this.alice);
+    }
+
+    @Test(description = "This tests add to cart for user Alice.")
+    public void addToCartAlice() {
+        for (int i = 0; i < itemsOfAlice.length; i++) {
+            this.cart.addToCart(itemsOfAlice[i]);
+        }
+        validateWebPage("2", this.cart.getNumberOfItems(), "Pet store cart content before checkout is " +
+                "not as expected");
+    }
+
+    @Test(description = "This tests checkout cart for user Alice.")
+    public void checkoutCartAlice() throws InterruptedException {
+        this.cart.checkout();
+        validateWebPage("$ 675.00", this.order.getOrderValue(), "Pet store cart content after checkout is " +
+                "not as expected");
+    }
+
+    @Test(description = "This tests sign out for user Alice.")
+    public void signOutAlice() {
+        this.signOut(this.alice);
+    }
+
+    @Test(description = "This tests sign in for user Bob.")
+    public void signInBob() throws InterruptedException {
+        this.signIn(this.bob);
+    }
+
+    @Test(description = "This tests check order for user Bob.")
+    public void checkOrdersBob() throws InterruptedException {
+        validateWebPage(noOrdersPlaced, this.order.getOrderValue(),
+                "Pet store cart content after checkout is not as expected");
+    }
+
+    @Test(description = "This tests sign out for user Bob.")
+    public void signOutBob() {
+        this.signOut(this.bob);
+    }
+
+    @Test(description = "This tests the termination of pet-store backend and frontend cells")
+    public void terminate() throws Exception {
+//        terminateCell(BACKEND_INSTANCE_NAME);
+//        terminateCell(FRONTEND_INSTANCE_NAME);
+    }
+
+    @AfterClass
+    public void cleanup() {
+        webDriver.close();
+        try {
```

Comment:
This is a empty try-catch block. Please check.

Reply:
Updated.